tools/dgeni/templates/class.template.html

```diff
@@ -6,6 +6,15 @@ <h4 class="docs-api-h4 docs-api-class-name">
 <p class="docs-api-class-description">{$ class.description | safe $}</p>
 {%- endif -%}
 
+{% if class.extendedDocs.length %}
+<p class="docs-api-class-extends-clauses">
+  <span class="docs-api-class-extends-label">Extends:</span>
+  {% for extendedDoc in class.extendedDocs %}
+    <span class="docs-api-class-extends-type">{$ extendedDoc.name $}</span>
```

Comment:
I think it would be better to show this immediately after the name, e.g.
```
MatVerticalStepper extends MatStepper
```

Reply:
I tried that and it didn't look that well in my opinion.

![image](https://user-images.githubusercontent.com/4987015/32115590-af2157a0-bb47-11e7-8ca9-6f1ee2469b95.png)


But maybe there will be some advanced styling to make this look better. Let me know if we should go that way.